woocommerce/woocommerce-admin
# Onboarding - business step: add more options in the competitors list and other fixes

Fixes #3658

Onboarding - Business Details step:
- Added the platforms: `Amazon`, `eBay`, `Etsy`, `Squarespace`, when the user selects that he's selling on another platform.
- Changed the copy: `How many products do you plan to sell?` to `How many products do you plan to display?`
- Added new param to `recorderEvent`
- Added a new input to save the platform's name, when the user selects `other`.


### Screenshots
![screenshot-onb-business1](https://user-images.githubusercontent.com/1314156/75795349-f0fb7680-5d50-11ea-8977-82a9b85a8346.png)
![screenshot-onb-business2](https://user-images.githubusercontent.com/1314156/75795338-ee991c80-5d50-11ea-8c06-1aec975356dd.png)


### Detailed test instructions:

- Go to the Onboarding's Business Details step (URL `/wp-admin/admin.php?page=wc-admin&step=business-details`)
- Check that the label `How many products do you plan to display?` is visible.

- Press the `Which platform is the store using?` dropdown menu. Check that the new options are available.
- Select the option: `Other`.
- An input should appear. Add a text inside.
- The text should be saved after `Continue` is pressed.
